packages/dapp-simple-exchange/contract/deploy.js

```diff
@@ -26,37 +24,32 @@ export default async function deployContract(homeP, { bundleSource, pathResolve
   // === AWAITING TURN ===
   // =====================
   
-  // 1. Assays and payments
-  const purse0P = homeP~.wallet~.getPurse('moola');
-  const purse1P = homeP~.wallet~.getPurse('simolean');
-  const assay0P = purse0P~.getAssay();
-  const assay1P = purse1P~.getAssay();
-  const payment0P = purse0P~.withdraw(900);
-  const payment1P = purse1P~.withdraw(900);
+  // 1. Issuers
+  const wallet = homeP~.wallet;
+  const [[pursePetname0], [pursePetname1]] = await wallet~.getPurses();
```

Comment:
getPurses() returns an array of purse presences. The purses have getCurrentAmount(), getAllegedBrand(), deposit() and withdraw(). What values will the two petnames get?

Reply:
Is this based on the wallet in master, or the changes to the wallet API that are also in a PR?

Reply:
Based on the the `wallet-revamp` PR #672 .  I'll leave this comment open until it lands and we're rebased.

Reply:
`getPurses()` is now analogous to `getIssuers()`.  It returns `[[purse0Petname, purse0], [purse1Petname, purse1]...]`.